mappings/net/minecraft/loot/LootManager.mapping

```diff
@@ -1,11 +1,11 @@
 CLASS net/minecraft/class_60 net/minecraft/loot/LootManager
 	FIELD field_20752 conditionManager Lnet/minecraft/class_4567;
-	FIELD field_970 suppliers Ljava/util/Map;
+	FIELD field_970 lootTables Ljava/util/Map;
```

Comment:
```suggestion
	FIELD field_970 tables Ljava/util/Map;
```

Reply:
Not sure, "loot tables" is clearer don't you think?

Reply:
This is in `LootTableManager` class, so I think `loot` is already implied.

Reply:
Fair, fixed